README.md

```diff
@@ -2,21 +2,34 @@
 
 A discord bot to send feed updates
 
-## Instalation
+## Commands
+
+The following commands are supported:
+
+| Command                   | Description                                                                                                       |
+| ------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `$feed-add <name> <link>` | This command adds a new feed to the current channel.                                                              |
+| `$feed-list`              | this command lists all feeds in the current server with the following information: id, name, channel id and link. |
+| `$feed-remove <id>`       | This command removes the feed addressed to the given id.                                                          |
+
+## Contributing
+
+### Instalation
 
 1. Create a `.env` file following the example file `.env.example`
 
 2. Install the dependencies
-    ```
-        $ npm install
-    ```
+
+   ```
+       $ npm install
+   ```
 
 3. Run the project
-    ```
-        $ npm start
-    ```
+   ```
+       $ npm start
+   ```
 
-## Contributing
```

Comment:
I think Contributing works better, because looking at the context of the page, if it is still named as standards, it will be kind of confusing

Reply:
hmmm, so we change the parent section? or let's keep duplicated?